com.unity.render-pipelines.universal/Shaders/Particles/ParticlesLitInput.hlsl

```diff
@@ -18,10 +18,19 @@ half _BumpScale;
 
 half _DistortionStrengthScaled;
 half _DistortionBlend;
+
+#if defined(_CLEARCOAT) || defined(_CLEARCOATMAP)
```

Comment:
could be more maintainable to have a _CLEARCOAT_ENABLED macro that encompasses both these, to avoid testing for both almost everywhere it's used.

Reply:
No, these are to different cases and are built as "standalone" keywords, so that the work like enums and not in combined way. This way we can eliminate an invalid state. It was specifically changed to this form in a past review.
But yeah it could be nicer, I suppose we could define a "meta" macro for ´defined(_CLEARCOAT) || defined(_CLEARCOATMAP)`.

_CLEARCOAT is used when the inputs are constants.
_CLEARCOATMAP is used when the input is a texture.

Reply:
I was only trying to suggest preventing the copy/paste of the tests everywhere by having:

#define _CLEARCOAT_ENABLED (_CLEARCOAT || _CLEARCOAT_MAP)

Sorry if that was unclear.

Reply:
I like that suggestion. 
Having `_CLEARCOAT_ENABLED` would be quite nice.

Reply:
I'm going to do a separate PR for moving the sampler functions into one file. I can fix that then. 
Let's not mix it into this PR and change even more files.

Reply:
Agreed